AnnoDesigner.Core/Models/IBounded.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Windows;
+
+namespace AnnoDesigner.Core.Models
+{
+    public interface IBounded
+    {
+        Point Position { get; set; }
+
+        Size Size { get; set; }
+    }
+
+    public static class Extensions
```

Comment:
This should be a method implemented as part of the interface I think, unless there's a specific reason you've done otherwise?

Reply:
It is extension method because most implementation would be exactly the same.

This would have been great candidate for default interface implementation, but .NET Framework 4.7.2 doesn't support it.

Reply:
Yeah - if we eventually move to .Net Core then that could be something to look at. The reasoning makes sense.